stdlib/testing/testing.flux

```diff
@@ -132,13 +140,24 @@ builtin assertEmpty : (<-tables: stream[A]) => stream[A]
 // introduced: 0.18.0
 // tags: tests
 //
-builtin diff : (
-        <-got: stream[A],
-        want: stream[A],
-        ?verbose: bool,
-        ?epsilon: float,
-        ?nansEqual: bool,
-    ) => stream[{A with _diff: string}]
+diff = (
+    got=<-,
+    want,
+    verbose=false,
+    epsilon=0.000001,
+    nansEqual=false,
+) =>
+    {
+        return
+            _diff(
+                got,
+                want,
+                verbose,
+                epsilon,
+                nansEqual,
+            )
+                |> yield(name: "error")
```

Comment:
Is there a reason this needs to be called `error`? I would find `diff` more informative.

Reply:
It may output a diff here, but since this marks the failure of a test I'd say `error` (or `assert`, `testfail` etc) is more descriptive of the `yield`. You could envision sending a plain error message here instead to indicate that the test failed.